Description: In debian, the gnuais.conf-example is installed in
  /usr/share/doc/gnuais/examples/. Since gnuais will copy this file into
  ~/.config/gnuais/config the first time it runs, this patch is needed.
  It basically just updates the references to that file in the source code.
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: gnuais/src/cfgfile.c
===================================================================
--- gnuais.orig/src/cfgfile.c
+++ gnuais/src/cfgfile.c
@@ -370,14 +370,14 @@ int read_cfgfile(char *f, struct cfgcmd
 					}
 					else {
 						hlog(LOG_NOTICE, "Using gnuais.conf-example as a starting point for ~/.config/gnuais/config...");
-						ret = cpfile(conf_home_folder,"/usr/share/doc/gnuais/gnuais.conf-example");
+						ret = cpfile(conf_home_folder,"/usr/share/doc/gnuais/examples/gnuais.conf-example");
 						if(ret == -1) hlog(LOG_ERR, "Could not copy configuration file to the home folder");
 						else hlog(LOG_NOTICE, "DONE creating configuration file (~/.config/gnuais/config). You should edit this file manually!");
 					}
 				}
 				else {
 					hlog(LOG_NOTICE, "Using gnuais.conf-example as a starting point for ~/.config/gnuais/config...");
-					ret = cpfile(conf_home_folder,"/usr/local/share/doc/gnuais/gnuais.conf-example");
+					ret = cpfile(conf_home_folder,"/usr/local/share/doc/gnuais/examples/gnuais.conf-example");
 					if(ret == -1) hlog(LOG_ERR, "Could not copy configuration file to the home folder");
 					else hlog(LOG_NOTICE, "DONE creating configuration file (~/.config/gnuais/config). You should edit this file manually!");
 				}
